### PR TITLE
Add ESLint and gitignore the dist/ directory

### LINKED ISSUE #5: A couple ESLint errors/warnings
```sh
$ npm run lint

> testpilot-ga@0.2.1 lint /Users/pdehaan/dev/github/mozilla/testpilot-ga
> eslint src


/Users/pdehaan/dev/github/mozilla/testpilot-ga/src/TestPilotGA.js
  26:11  error    'x' is assigned a value but never used  no-unused-vars
  31:19  error    'options' is defined but never used     no-unused-vars
  39:7   warning  Unexpected console statement            no-console

✖ 3 problems (2 errors, 1 warning)
```

Where my .eslintrc.yml file is:

```yml
env:
  browser: true
  es6: true
  node: true

extends:
  - eslint:recommended

parser: babel-eslint

root: true

rules:
  eqeqeq: error
  no-console: warn
  no-var: error
  prefer-const: error
```

And here's my diff:

```diff
diff --git a/package.json b/package.json
index efec8b3..a7d90e5 100644
--- a/package.json
+++ b/package.json
@@ -9,22 +9,26 @@
   "devDependencies": {
     "babel-cli": "^6.24.1",
     "babel-core": "^6.25.0",
+    "babel-eslint": "^7.2.3",
     "babel-plugin-add-module-exports": "^0.2.1",
     "babel-plugin-transform-class-properties": "^6.24.1",
     "babel-plugin-transform-es2015-modules-umd": "^6.24.1",
     "babel-preset-env": "^1.6.0",
+    "eslint": "^4.2.0",
     "husky": "^0.14.3",
     "lint-staged": "^4.0.2",
     "prettier": "^1.5.3"
   },
   "scripts": {
     "build": "mkdir -p dist && babel src/TestPilotGA.js --out-file dist/index.js",
+    "lint": "eslint src",
     "precommit": "lint-staged && npm run build",
     "publish": "npm run build && npm publish"
   },
   "lint-staged": {
     "*.js": [
       "prettier --write",
+      "eslint",
       "git add"
     ]
   }
```

### LINKED ISSUE #6: Add dist/ to .gitignore
Or whatever. Not like it's at risk of getting bundled into the published npm module.

### PR DESCRIPTION
**Pull request by pdehaan**
Fixes #5 
Fixes #6

### CODE REVIEW

File: src/TestPilotGA.js [hunk 1]
```diff
@@ -23,7 +23,6 @@ export default class TestPilotGA {
   }
 
   setOptions(options) {
-    const x = 1;
```
Comment (chuckharmston) [inline src/TestPilotGA.js]: I don't know how this got there `o.O`

File: .eslintrc.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+env:
+  browser: true
+  es6: true
+  node: true
```
Comment (chuckharmston) [inline .eslintrc.yml]: Can you add `webextensions: true` please?

Comment (pdehaan) [inline .eslintrc.yml]: Done!

**Comment by pdehaan** [comment]
Ah, it doesn't look like this repo has Travis or Circle-CI.

But if you run `npm run lint`, you should get one error:

```sh
 $ npm run lint

> testpilot-ga@0.2.1 lint /Users/pdehaan/dev/github/mozilla/testpilot-ga
> eslint src

/Users/pdehaan/dev/github/mozilla/testpilot-ga/src/TestPilotGA.js
  30:19  error  'options' is defined but never used  no-unused-vars

✖ 1 problem (1 error, 0 warnings)
```

I'm not sure how to address that, apart from removing the `options` argument from the `validateOptions()` method:

```js
validateOptions(options) {
```

**Pull request merged**